config.toml

```diff
@@ -274,4 +274,8 @@ disableAliases = true
     weight = 5
 
 [permalinks]
-  case-studies = "/:slug"
\ No newline at end of file
+  case-studies = "/:slug"
+
+[security]
+  [security.funcs]
+    getenv = ['^HUGO_','URL']
```

Comment:
The updated hugo complains 

> Error: Error building site: failed to render pages: render of "home" failed: "/work/layouts/rss.xml:1:45": execute of template failed: template: rss.xml:1:45: executing "rss.xml" at <os.Getenv>: error calling Getenv: access denied: "URL" is not whitelisted in policy "security.funcs.getenv"; the current security configuration is:

See https://github.com/istio/istio.io/blob/c702c2d96f53bae4798cac8578c790b0d96c9588/layouts/rss.xml#L1